<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoModeWindow</class>
 <widget class="QWidget" name="AutoModeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>391</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="deviceTable"/>
    </item>
    <item>
     <widget class="QTableView" name="sourceTable"/>
    </item>
    <item>
     <widget class="QTableView" name="bufferTable"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="labelForTime">
      <property name="text">
       <string>Время события:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditForTime"/>
    </item>
    <item>
     <widget class="QLabel" name="labelForStep">
      <property name="text">
       <string>Шаг:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditForStep"/>
    </item>
    <item>
     <widget class="QLabel" name="labelForAllSteps">
      <property name="text">
       <string>Всего событий:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditForAllSteps"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>391</width>
     <height>112</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="labelForEvent">
      <property name="text">
       <string>Описание события:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowserForEvent"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="labelForOtherStep">
      <property name="text">
       <string>Перейти на шаг:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditForOtherStep"/>
    </item>
    <item>
     <widget class="QPushButton" name="buttonForOtherStep">
      <property name="text">
       <string>Перейти</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>800</y>
     <width>389</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="backStepButton">
      <property name="text">
       <string>&lt; Предыдущий шаг</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="nextStepButton">
      <property name="text">
       <string>Следующий шаг &gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="backButton">
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
